Extensions/quick_tags.js

```diff
@@ -412,8 +413,8 @@ XKit.extensions.quick_tags = new Object({
 
 		$posts
 			.addClass("xkit-quick-tags-done")
-			.each(function() {
-				XKit.interface.react.add_control_button($(this), "xkit-quick-tags", "");
+			.each(async function() {
+				await XKit.interface.react.add_control_button($(this), "xkit-quick-tags", "");
```

Comment:
surely this doesn't actually change anything? there's only one line of code in this function, so using `await` here does nothing - async functions are async, so `.each()` won't wait for each one to complete before running the next one

Reply:
yeah, it's not. was just in there from initial fiddling, shouldn't have been up.